strawberry/schema/schema.py

```diff
@@ -177,11 +178,7 @@ def execute_sync(
         if result.errors:
             self.process_errors(result.errors, execution_context=execution_context)
 
-        return ExecutionResult(
-            data=result.data,
-            errors=result.errors,
-            extensions=result.extensions,
-        )
+        return result
```

Comment:
the type of `result` is not Strawberry's `ExecutionResult`, I think we should revert this change 😊

Reply:
![image](https://user-images.githubusercontent.com/4014888/129641340-f5a285b0-a982-4d6f-9c5f-55111671ae7e.png)

https://github.com/strawberry-graphql/strawberry/blob/803cad533ed1a265117a3d3800265cba549e81c1/strawberry/schema/execute.py#L107

At the beginning they rename `ExecutionResult` from `graphql` to `GraphQLExecutionResult`
https://github.com/strawberry-graphql/strawberry/blob/803cad533ed1a265117a3d3800265cba549e81c1/strawberry/schema/execute.py#L7

Reply:
I think @scratchmex is right @patrick91 . We're already returning the same type from `schema.execute.execute_sync`. @scratchmex can you change the return value of the `execute` method as well please?

Reply:
oops, my bad!